.github/CODEOWNERS

```diff
@@ -7,12 +7,12 @@
 # Cryptography:
 crypto/* @darkeld3r @aleksuss @pavel-mukhanov @vldm @slowli
 
-# Cryptocurrency examples:
-examples/cryptocurrency/* @darkeld3r @aleksuss @vitvakatu @ilammy
-examples/cryptocurrency-advanced/* @darkeld3r @aleksuss @vitvakatu @ilammy
+# Demo service examples:
```

Comment:
Just for curiosity, why not `demo-cryptocurrency` ? `demo-service` it's abstract name as for me.

Reply:
Why not `cryptocurrency-demo`? :D

Reply:
I understand that the PR might seems a bit strange, but its an attempt at making names of demos uniform and more user-friendly across docs and in the code.
In this case, the motivation was that cryptocurrency is a rather hyped up word, and it might make the wrong impression. We did not want the name to be instantly associated with ICO, etc., but rather with what it is - a demo showing how to implement a basic service using Exonum.
Hope I managed to explain the case a bit.